.github/workflows/mediasoup-worker.yaml

```diff
@@ -69,8 +69,8 @@ jobs:
 
       - run: npm install
       - run: |
-          pushd worker/scripts
+          cd worker/scripts
           npm install
-          popd
+          cd ../..
```

Comment:
I think it runs every command under the same directory regardless of `cd` in previous command...

Reply:
oh, makes sense. Thanks

Reply:
So probably this is executing `npm install` in the root directory again instead of in `worker/sripts`, right?

Reply:
Fixed, let's see